{"name": "test_buurten_failure", "status": "passed", "description": "Failure API request for the buurten", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1646831788175, "stop": 1646831788317}], "start": 1646831788175, "stop": 1646831788317, "uuid": "44a1c60f-2146-47ef-ac0c-f11bef29a62e", "historyId": "6d2d2afe8521e49849a7d1201a789880", "testCaseId": "76f46efd1f9a250d2bb8382d9d249708", "fullName": "e2etest.test_datasetgebiedenbuurten#test_buurten_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenbuurten"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "55159-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenbuurten"}]}